presto-orc/src/main/java/io/prestosql/orc/metadata/OrcMetadataWriter.java

```diff
@@ -268,16 +269,18 @@ private static UserMetadataItem toUserMetadata(Entry<String, Slice> entry)
     public int writeStripeFooter(SliceOutput output, StripeFooter footer)
             throws IOException
     {
-        OrcProto.StripeFooter footerProtobuf = OrcProto.StripeFooter.newBuilder()
+        OrcProto.StripeFooter.Builder footerProtobuf = OrcProto.StripeFooter.newBuilder()
                 .addAllStreams(footer.getStreams().stream()
                         .map(OrcMetadataWriter::toStream)
                         .collect(toList()))
                 .addAllColumns(footer.getColumnEncodings().stream()
                         .map(OrcMetadataWriter::toColumnEncoding)
-                        .collect(toList()))
-                .build();
+                        .collect(toList()));
 
-        return writeProtobufObject(output, footerProtobuf);
+        footer.getTimeZone().ifPresent(zone ->
```

Comment:
isn't `footer.getTimeZone()` always present for the Presto writer?  If, so I would just assert that on the first line and then unconditionally, add the TZ.

Reply:
Good idea